File: src/main/java/com/example/kotrip/dto/daytrip/NewNode.java
package com.example.kotrip.dto.daytrip;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class NewNode {
    private Long startId;
    private String startName;
    private Double startLatitude;
    private Double startLongitude;
    private Long destId;
    private String destName;
    private Double destLatitude;
    private Double destLongitude;
}

File: src/main/kotlin/com/example/kotrip/dto/NewNode.kt
package com.example.kotrip.dto

data class NewNode(
    val startId: Int,
    val startName: String,
    val startLatitude: Double,
    val startLongitude: Double,
    val destId: Int,
    val destName: String,
    val destLatitude: Double,
    val destLongitude: Double,
    val time: Int
)
